apps/src/code-studio/responsiveRedux.js

```diff
@@ -0,0 +1,37 @@
+import { makeEnum } from '@cdo/apps/utils';
+
+const SET_RESPONSIVE_SIZE = 'responsive/SET_RESPONSIVE_SIZE';
+export const setResponsiveSize = responsiveSize => ({ type: SET_RESPONSIVE_SIZE, responsiveSize });
+
+export const ResponsiveSize = makeEnum('lg', 'md', 'sm', 'xs');
+
+// Default window widths that are the starting points for each width category.
+const Breakpoints = [
```

Comment:
Has any further thought gone into these widths? I remember @breville choosing ones that worked well for the page he worked on. 992 matches bootstrap, but I'm not sure where the other ones come from: https://v4-alpha.getbootstrap.com/layout/overview/#responsive-breakpoints

Reply:
Since our page size width is [970](https://github.com/code-dot-org/code-dot-org/blob/staging/shared/css/style-constants.scss#L14) I'd expect one of the breakpoints to be like 972 or something?